src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -50,6 +50,7 @@
     CONFIG_NAME,
     DEPRECATED_REVISION_ARGS,
     BaseOutput,
+    DDUFReader,
```

Comment:
TODO: take this from https://github.com/huggingface/huggingface_hub/pull/2692/

Reply:
switched !